Core/Main/itkTransformixFilter.h

```diff
@@ -103,6 +103,8 @@ class ITK_TEMPLATE_EXPORT TransformixFilter : public ImageSource<TMovingImage>
 
   using MeshType = Mesh<OutputImagePixelType, MovingImageDimension>;
 
+  using TransformType = Transform<double, MovingImageDimension, MovingImageDimension>;
```

Comment:
Is it OK for the CombinationTransform to assume that FixedImageDimension == MovingImageDimension?